### ISSUE TITLE
author data?

### ISSUE THREAD

**Issue by ashleygwilliams** [ashleygwilliams/wasm-pack#2]
we can grab author data from Cargo as well, but we MIGHT consider checking if there is an `.npmrc` and preferring that if it exists

**Comment by sendilkumarn** [comment]
Do you think creating a npmrc crate makes sense here? to fetch some information from npm related files or do we already have one?

**Comment by ashleygwilliams** [comment]
npmrc crate- it's possible. what would you envision it doing? i can spin something up like that pretty quick. i'll play around with it today!

**Comment by sendilkumarn** [comment]
yup basically, think it will be more useful if we want to fetch some more information from the npmrc or anyother npm based files in future.

**Comment by yoshuawuyts** [comment]
Took a quick stab at creating a crate specifically for this. https://github.com/yoshuawuyts/npmrc. Added you both as collaborators, so if there's anything not working any one of us can easily fix it!

Usage should be reasonably straight forward:
```rust
extern crate npmrc;
let npmrc_values = npmrc::read()::unwrap();
println!("{:?}", npmrc_values);
```

Under the hood it's using [serde-ini](https://github.com/arcnmx/serde-ini) for the parsing. I haven't quite gotten enums to work as expected tho, and like by no means are we parsing [all of npm's possible values](https://docs.npmjs.com/files/npmrc). But feel it's a start, and hope it comes in helpful!

**Comment by sendilkumarn** [comment]
I guess, first cut we should just extract the authors and then concentrate on making this crate to parse all the available information.

WDYT?

**Comment by ashleygwilliams** [comment]
i'm working on implementing this rn! PR incoming

**Comment by ashleygwilliams** [comment]
https://github.com/ashleygwilliams/wasm-pack/pull/11

**Comment by ashleygwilliams** [comment]
closed by #70
